game-core/src/main/java/tools/image/TileImageReconstructor.java

```diff
@@ -61,14 +60,14 @@ private TileImageReconstructor() {}
    *
    * @throws IllegalStateException If not invoked on the EDT.
    */
-  public static void run(final String[] args) {
+  public static void run() {
     checkState(SwingUtilities.isEventDispatchThread());
 
-    new TileImageReconstructor().runInternal(args);
+    new TileImageReconstructor().runInternal();
   }
 
-  private void runInternal(final String[] args) {
-    handleCommandLineArgs(args);
+  private void runInternal() {
```

Comment:
Codacy found an issue: [The method 'runInternal()' has an NPath complexity of 1944, current threshold is 200](https://app.codacy.com/gh/triplea-game/triplea/pullRequest?prid=6179649)